docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -281,63 +281,98 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
     * This should be ``main`` in the case of an unspecified entry point.
     * This will be the base name of the tasks file to execute, with no ``.yml`` or ``.yaml`` file extension.
 
-    :short_description:
+.. list-table:: Role Argument Specification Fields
+  :width: 90%
+  :widths: auto
+  :header-rows: 1
+
+    * - Name
+      - Description
+      - Type
+      - Required
+      - Default
+    * - short_description
+      - A short, one-line description of the entry point.
+      - str
+      - Yes
+      -  
+    * - description
+      - A longer description that may contain multiple lines.
+      - str
```

Comment:
antsibull-docs only accepts lists of strings here, same as for modules and plugins. Since ansible-doc also accepts lists of strings I would at least mention that these are supported as well, or even just mention lists of strings here.